### PR TITLE
Style update: `Navbar` component

### LINKED ISSUE #127: Style Audit: `Navbar` Style Update
# Description

Currently, the bottom navigation component we use doesn't stylistically match the rest of the site. It uses blue buttons and a font that isn't used anywhere else.

Please update this component to match the design language that we've established for the rest of the site.

# Acceptance Criteria

- [ ] Update the buttons on the `Navbar` to use the same color of orange that's used throughout the rest of the site.
- [ ] Make sure the font is either Roboto or Open Sans (see #126)

Feel free to make other changes that would either increase the contrast of the buttons or the `Navbar` itself. If you're ensure about the design, ask in your PR and we'll assess it.

# Help

Here are the docs for MUI's `BottonNavigation` component, which `Navbar` uses under the hood: https://mui.com/material-ui/react-bottom-navigation/

### PR DESCRIPTION
**Pull request by trevortippery**
## Description
This pull request updates the style of the `navbar` component that is used at the bottom of the site to match the design established throughout the rest of the site. 

## Related Issue
This pull request closes #127 and addresses #126. 

## Type of Changes
Please select the relevant option:
- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Documentation update
- [ ] Other (please specify)
      
The update includes matching the same color of orange that is used throughout the rest of the site, setting the font of the labels to Roboto or Open Sans, and changing the font color of the labels to black so it matches the theme of the site.

## Things to Update
Please add any features, buttons, or pages that might need updating in the future:
- [ ] Button Functionality 
- [ ] Code Documentation 
- [ ] Mobile Responsiveness 
- [ ] Code testing 
- [ ] Other (please specify)

## Screenshots 
Screenshot 2024-04-23 at 10 43 36 AM

## Additional Notes
I utilized issue #126 as a reference while implementing the font family Roboto or Open Sans. To achieve this, I directly imported the `index.css` file into the SCSS module of the `navbar` component. This approach eliminated the necessity for creating a new class specifically for overriding the `font-family` attribute. This pull request can wait to be squashed and merge until issue #126 is resolved.

### CODE REVIEW

File: client/src/components/Navbar/Navbar.module.scss
Comment (KingJordan152) [inline client/src/components/Navbar/Navbar.module.scss:1]: I understand your justification, but I don't think this will be necessary once #126 is merged in. That issue should change the font family project-wide without having to do any extra work within the components themselves. If that's not the case, then you should change the `font-family` directly, not import the index file.

**Review by KingJordan152 — CHANGES_REQUESTED** [review]
Overall, great work on this issue. I think the `Navbar` looks a million times better since it now aligns with our design system. 

I have a small issue regarding your use of the `index.css` file, but we can address that once #126 has been finished. I'll come back and approve this afterwards, as well.

File: client/src/components/Navbar/Navbar.module.scss
Comment (trevortippery) [reply client/src/components/Navbar/Navbar.module.scss:1]: @KingJordan152 I implemented #126 changes on my local environment and it still works without importing the `index.css`. I kept it in for now and will delete it once #126 is resolved. It just creates a duplicated `:root` attribute.

**Review by Aligary — APPROVED** [review]
Great improvement to our visuals!

**Review by dafneMena — APPROVED** [review]
Aside from Jordan's comment everything else looks good, nice work!

**Comment by KingJordan152** [comment]
Could you also lift the`Navbar` icons up a bit in this PR? Currently it's being blocked by the app slide-up bar when in mobile PWA form.

image

File: client/src/components/Navbar/Navbar.module.scss
Comment (trevortippery) [reply client/src/components/Navbar/Navbar.module.scss:1]: The changes were made.

**Comment by trevortippery** [comment]
I added `2 rem` of padding on the bottom of the `navbar` component.

**Review by KingJordan152 — APPROVED** [review]
This looks great, thank you for implementing my requests, especially the last one about the `Navbar` padding. Approved!

**Pull request merged**
